Add Basque localisation

xabirequejo opened the pull request:
Hi,
I believe I have translated all the files and most of the strings for Buran to be usable in Basque.
It would be great if Basque locale was added to Buran in a future update.
Thanks

xabirequejo committed:
Create full_description.txt

xabirequejo committed:
Create short_descripction.txt

xabirequejo committed:
Create title.txt

xabirequejo committed:
strings for Basque

Hasierako bertsioa / Initial localisation

Corewala commented:
Hi, this is really amazing work, thank you so much! I noticed that "show_inline_images" and "pkcs_notice" aren't translated yet, but other than that everything looks good! Since I'm going to work on a couple other bugfixes before the next release, I'll keep this PR open for now, but I'll definitely merge it before v1.13!

Corewala commented:
Because of https://github.com/Corewala/Buran/commit/df6c159c2cf83f6c9ecabc053b2e978c660a6141 you don't need to translate "load_image" since it isn't used anywhere.

xabirequejo commented:
Truth is I have no idea how to translate `pkcs_notice` so I thought leaving it as is was the better option.
I'll edit the file to include `inline_images` and remove `load_image`.

Could _PKCS12 client keystores_ be translated as ≈ password wallets or managers?

xabirequejo committed:
Updated EU strings

- 'load_image' has been removed
- 'inline_icons' & 'inline_images' have been translated
- 'pkcs_notice' might be translated
- small fixes

Corewala commented:
> Could PKCS12 client keystores be translated as ≈ password wallets or managers?

Honestly I was considering changing the wording for that string since elsewhere they're called "client certificates." Would "Only PKCS12 certificate files are currently supported" be easier to translate?

xabirequejo commented:
Yes, that would make it a lot easier. So long as it's still accurate.
≈ PKCS12 ziurtagiri-fitxategiak soilik erabil daitezke oraingoz.

Corewala commented:
Yes, that should be accurate, and honestly probably more informative than "client keystores."
